extension-manifest-v3/src/background/env.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+if (!chrome.webRequest) {
```

Comment:
As we are not in a rush, can we fix the issue in a final way, without monkey-patching the chrome API?

Reply:
Oh, actually good point. Monkey patching makes also feature detection of APIs difficult (or impossible?). For page messages, I'm assuming that it is possible to detect missing APIs. Monkey patching can break assumptions in other places.

Reply:
it is not easy to fix it the right way. having this merged and tested on a platform that does not resolve to monkey-patching is IMAO way more important.

Reply:
I'm just realizing that it is in the wrapper, not the library itself. I would be fine with leaving it for now as is. Ideally, we should eventually change the library code to expect missing APIs. I think, then we can safely remove the monkey patching.

(I also checked: it will not conflict with current assumptions in page messages.)